### PR TITLE
added multichart capabilities

### PR DESCRIPTION
**Pull request by virtustate**

### CODE REVIEW

File: nvd3/multiChart.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+class multiChart(TemplateMixin, NVD3Chart):
+
+    """
+    A line chart or line graph is a type of chart which displays information
+    as a series of data points connected by straight line segments.
+
+    Python example::
+
+        from nvd3 import lineChart
```
Comment (areski) [inline nvd3/multiChart.py]: The inlines docs needs to be updated to reflect multiChat also the example and the output

File: nvd3/multiChart.py [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+        chart.add_serie(y=ydata, x=xdata, name='sine', extra=extra_serie, **kwargs1)
+        extra_serie = {"tooltip": {"y_start": "", "y_end": " min"}}
+        chart.add_serie(y=ydata2, x=xdata, name='cose', extra=extra_serie, **kwargs2)
+        chart.buildhtml()
+
+    Javascript renderd to:
+
+    .. raw:: html
+
+        <div id="lineChart"><svg style="height:450px; width:100%"></svg></div>
```
Comment (areski) [inline nvd3/multiChart.py]: still refering to linechart

**Comment by virtustate** [comment]
exampes/multiChart.py added and nvd3/multiChart.py modified to contain example

File: nvd3/__init__.py [hunk 3]
```diff
@@ -26,5 +27,5 @@
 from .scatterChart import scatterChart
 from .discreteBarChart import discreteBarChart
 from .multiBarChart import multiBarChart
-from .bulletChart import bulletChart
```
Comment (areski) [inline nvd3/__init__.py]: You removed bulletChart? any reason?

Comment (virtustate) [reply nvd3/__init__.py]: Sorry, a careless mistake, added back in.

**Comment by areski** [comment]
Thanks for the efforts, the test are now working!

**Pull request merged**